CHANGELOG-prerelease.md

```diff
@@ -8,6 +8,7 @@ Tag: [v1.0.0-prerelease.28](https://github.com/patternfly/patternfly-elements/re
 - [5800171](https://github.com/patternfly/patternfly-elements/commit/5800171f9f6afc95f832a01bce7f4eeb5c6717d7) fix: set pfe-band to private, until frameworks + ie11 support (#589)
 - [a4b2c27](https://github.com/patternfly/patternfly-elements/commit/a4b2c2751840265831329729ed577651b7099388) fix: build tasks and packages to represent new dist architecture
 - [](https://github.com/patternfly/patternfly-elements/commit/) pfe-navigation style updates
```

Comment:
Be a good citizen Kyle - update the line item before yours with the commit hash!  That way I'm not the one doing it for 10 of them at once :P

Reply:
Updated again. Sorry.